sound_cloud_backend/manage.py

```diff
@@ -2,7 +2,8 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
-
+from dotenv import load_dotenv
```

Comment:
Its fine to use dotenv for environment variables. Another option is python-decouple; if you go with the latter option, you don't have to write these two lines neither in manage.py nor in wsgi.py.